packages/components/select/src/select.vue

```diff
@@ -191,6 +192,7 @@
             :validate-event="false"
             :class="[nsSelect.is('focus', visible)]"
             :tabindex="multiple && filterable ? -1 : undefined"
+            :aria-label="ariaLabel"
```

Comment:
```suggestion
            :label="ariaLabel"
```

[ElInput label](https://element-plus.org/en-US/component/input.html#attributes)

At the same time, can you update the document?

Reply:
Simple html `input` doesn't have a `label` attribute, and if I change that - the accessibility issue comes back.

Reply:
You can take a look at the ElInput component's documentation or source code

Reply:
if you're talking about [this](https://github.com/element-plus/element-plus/blob/dev/packages/components/input/src/input.vue), `input` has an `aria-label` attribute, not `label`

Reply:
Or are you asking me to just change `label` to `aria-label` in [this documentation](https://element-plus.org/en-US/component/input.html#attributes)? And change nothing in my code

Reply:
Due to some historical reasons, the `aria-label` attribute is configured through the `label` attribute in the ElInput component. It cannot be modified in a short period of time. So I think we should use the `label` to insert parameters into the ElInput component here.

Reply:
Add a new attribute should be displayed in the documentation

Reply:
[Fixed ](https://github.com/element-plus/element-plus/pull/13932/commits/bcdf7d6f373d112d8c34543d2e98a9eadb7a1f3a)